CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+v0.6.0
+======
+
+In #204 :
```

Comment:
Could you use the same format as for other versions (PR number at the end of each line) please?